.travis.yml

```diff
@@ -1,14 +1,30 @@
 language: csharp
 
+sudo: required
+dist: trusty
+
+env:
+  - CLI_VERSION=1.0.0-preview1-002702
+
+addons:
+  apt:
+    packages:
+    - gettext
+    - libcurl4-openssl-dev
+    - libicu-dev
+    - libssl-dev
+    - libunwind8
+
```

Comment:
Not that this is the cause of the test failure, but I will assume that those dependencies are required by some parts of the runtime one way or another, so eventually they should be added to the setup.